### DISCUSSION TITLE
Ways to create movement scripts?

### DISCUSSION THREAD

**Discussion by pmcthor777** [kinsi55/CS_BeatSaber_Camera2#23] [Q&A] [answered]
What is the best way to create a movement script? I've never created one and  want to learn. Is modifying the .json file the only way to make one?

**Comment by kinsi55** [comment, marked as answer]
Yes, currently the only way to create movement scripts is manually. I had planned to possibly eventually create a UI for editing / creating movement scripts but that is very low priority - I dont think such a thing exists for camera+ either, but if it does you could use that and just import the camera+ movement script.

**Reply by pmcthor777** [reply]
OK, thanks for the reply!
